src/App.css

```diff
@@ -11,7 +11,8 @@
     display: flex;
     flex-direction: column;
     padding: 20px;
-    background-color: #F2C6C2;
+    /* background-color: #F2C6C2; */
+    background-color: #F2E8DF;
     color: #171212;
     border-radius: 10px;
 }
```

Comment:
Your media query on `screen` works but there is no element with class `.container` to which we can apply that style when we hit 800px breakpoint! I think you probably want either `#root` or `.appContainer`. When I switch to `.appContainer { width: 80%; }` it still looks a bit weird, but `75vw` makes it more clear that it's changing.